pkg/sdn/plugin/subnets.go

```diff
@@ -107,6 +107,17 @@ func (master *OsdnMaster) deleteNode(nodeName string) error {
 	}
 
 	log.Infof("Deleted HostSubnet %s", hostSubnetToString(sub))
+
+	_, sn, err := net.ParseCIDR(sub.Subnet)
+	if err != nil {
+		return fmt.Errorf("error parsing network address %q: %v", sub.Subnet, err)
+	} else {
+		err := master.subnetAllocator.ReleaseNetwork(sn)
```

Comment:
@liggitt  Actually ReleaseNetwork failed only when sn is a wrong subnet, which means there is a bug in some code. So here we just send the error message out to log file to indicate the issue.